### DISCUSSION TITLE
Make method that allows comparison of estimates from a number of different samplers / runs

### DISCUSSION THREAD

**Discussion by ben18785** [pints-team/pints#1638] [Ideas/Science]

**Comment by MichaelClerx** [comment]
I like this!
Do you intend to add some kind of module (a bit like `pints.diagnostics`) that can do things like estimate KL divergence (or Kolmogorov Smirnov or similar?)

**Comment by MichaelClerx** [comment]
There's some things like this already:
- https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.normaltest.html#scipy.stats.normaltest
- https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.kstest.html#scipy.stats.kstest
- https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.anderson_ksamp.html#scipy.stats.anderson_ksamp
